GVFS/GVFS.UnitTests/Common/NuGetUpgrade/NuGetUpgraderTests.cs

```diff
@@ -54,20 +66,27 @@ public void SetUp()
                 this.downloadDirectoryPath,
                 null,
                 this.tracer);
+
             this.mockFileSystem = new MockFileSystem(
                 new MockDirectory(
                     Path.GetDirectoryName(this.downloadDirectoryPath),
                     new[] { new MockDirectory(this.downloadDirectoryPath, null, null) },
                     null));
 
+            this.mockCredentialManager = new Mock<ICredentialStore>();
```

Comment:
What are our standards with using Mock?  I have always used [`MockBehavior.Strict`](https://github.com/moq/moq4/wiki/Quickstart) so that you know exactly what the expected calls for your test are.

Reply:
> What are our standards with using Mock? I have always used MockBehavior.Strict so that you know exactly what the expected calls for your test are.

I don't think we currently have any, but after reading about `MockBehavior.Strict` it sounds like a good guideline.

Unless someone has objections I think it makes sense to update our current `Mock` code to use it (if we're not already) and adding it to the coding guidelines (either in this PR or a separate PR).

Reply:
> either in this PR or a separate PR)

Let's do this in a separate PR.